### ISSUE TITLE
Enable new Users to be associated with existing Profiles

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network-pulse-api#338]
**Situation**: For many fellows, we have created their profiles for them. For those people to update their own profile and start to use Pulse, we need a means to manual add them. In another universe, everyone would join pulse first, then become a fellow. That is not our reality.

Let's discuss @gideonthomas @alanmoo 

### A) Ideal workflow
 1. person is named on some projects. they later apply and become a fellow. 
 1. person becomes a fellow or mozfest speaker.
 1. we create profile associated with projects.
 1. we initiate a user account (with permission) using email address and associate with profile
 1. they login with gauth, and presto, it works

**Note**: Currently, `Add User` seems to be disabled in django. I haven't been able to test steps 4 and 5. 

### B) Less than ideal workflow
 1. person named on projects
 1. person becomes a fellow
 1. we create profile assoc with projects
 1. person joins pulse
 1. we associate account with profile, and delete the profile that's auto-created with a new user

cc @annmarie123

**Comment by xmatthewx** [comment]
Yes. An staff_admin merge feature would solve this fairly well. It matches B above, but simplifies the last step. Though we'd have to decide which profile to favor in the merge. 

Option A means that things would work on first use by the person, set up by staff in advance.

**Comment by gideonthomas** [comment]
Ah I see. If I understand correctly, option A is more of a process-level change than a system-level change? the third step (for option B too) will become a non-issue once my patch on replacing creators with profiles lands. 4 and 5 are probably what we need to test out to make sure they work.

**Comment by xmatthewx** [comment]
Yes. One thing I didn't make totally clear above... this doesn't work: https://pulse-api.mofostaging.net/admin/users/emailuser/add/. Might be a bug, or the team might have disabled that on purpose.

**Comment by gideonthomas** [comment]
I wouldn't be surprised if we disabled it since it's tied to gauth and might be a little complicated to create users.

**Comment by alanmoo** [comment]
I’d bet that’s it too. I don’t think we’re anywhere near the ability to pre-create user accounts.

**Comment by xmatthewx** [comment]
@annmarie123 - setting up fellows will have to be a 2 step process for them unfortunately.  1) Login to Pulse (and let you know they've done it). 2) You link their account and profile (and tell them it's ready).

We still need to enable this linking in the admin. 

@gideonthomas - should we file a new ticket to clearly scope the work we plan to do?

**Comment by annmarie123** [comment]
Thanks, @xmatthewx for researching all possibilities! I'll make a note of the process you mentioned in the pulse FAQ doc for alumni.

**Comment by xmatthewx** [comment]
Is the advantage of merge that we can gather all associated entries from both profiles? How will we merge profile field content?

**Comment by gideonthomas** [comment]
Yeah it's mainly collation. We would need to think about the UX of merge - it's not as simple as click a button and it's done, we could add something like conflict resolution.

**Comment by xmatthewx** [comment]
That sounds complicated, but useful. I'll close this and we can talk about merge stuff in #327. 

@annmarie123 feel free to follow the thread there.
